### PR TITLE
Pr/work on issue 1317

### PR DESCRIPTION
**Pull request by sdeseille**
Hello

I added documentation about serializer configuration in Dancer2::Core::Role::Serializer module and in Dancer2::Config.pod.

I hope to haven't done too much mistake.

regards

sdeseille

### CODE REVIEW

File: lib/Dancer2/Config.pod [hunk 1]
```diff
@@ -227,6 +227,54 @@ extension, and are text files, set the C<default_mime_type> to
 C<text/plain>.
 
 
+=head2 Serializing responses
+
+=head3 serializer (string)
```
Comment (sdondley) [inline lib/Dancer2/Config.pod]: I can't speak to the content of the patch, so I'll be a doc Nazi and suggest that `(string)` should probably be replaced with something more perlish like ($string). Thanks!

Comment (sdeseille) [reply lib/Dancer2/Config.pod]: hello, it is really (string) and not $string.

You have to understand (string) has a datatype like per example (integer).
regards

sdeseille

**Comment by racke** [comment]
Indeed, that is consistent with the rest of the POD. This change looks good to me 👍

**Comment by cromedome** [comment]
:+1:

Merged, thank you!

**Pull request closed without merging**